### DISCUSSION TITLE
[Core Protocol] Does OP Stack L2 chain supports threshold BLS signatures?

### DISCUSSION THREAD

**Discussion by traderzk** [ethereum-optimism/developers#391] [Core Protocol] [answered]
### Did you check the documentation?

- [X] Yes

### Did you read the specs?

- [X] Yes
- [ ] No

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

we plan to deploy a L2 chain base on OP Stack, but we want to know if the chain supports threshold BLS signatures?

**Comment by hash1go** [comment, marked as answer]
Hello!
OP Stack does not support threshold BLS signatures. It only supports ECDSA signatures.

**Reply by traderzk** [reply]
ok, so if there is a need for threshold BLS signatures. it can implement through smart contracts on the chain, right?

**Reply by hash1go** [reply]
Theoretically, it is possible to deploy a contract to verify threshold BLS signatures, but it depends on the specifications.
At the very least, OP Stack does not support threshold BLS signatures without modifications to the code.

**Reply by traderzk** [reply]
alright, get it. thank you

**Reply by hash1go** [reply]
If it has been resolved, you can close this discussion.
